api/services/events/v1/events.proto

```diff
@@ -3,17 +3,23 @@ syntax = "proto3";
 package containerd.services.events.v1;
 
 import "gogoproto/gogo.proto";
-import "google/protobuf/timestamp.proto";
 import "google/protobuf/any.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
 
 option go_package = "github.com/containerd/containerd/api/services/events/v1;events";
 
 service Events {
 	rpc Stream(StreamEventsRequest) returns (stream Envelope);
+	rpc Post(PostEventRequest) returns (google.protobuf.Empty);
```

Comment:
Briefly consider that we may want to separate these into publisher and subscriber. That use case may not make sense.